VirtoCommerce.Platform.Web/Scripts/app/dynamicProperties/blades/propertyValue-list.js

```diff
@@ -39,6 +41,15 @@
         blade.isLoading = false;
     };
 
+    function refreshDynamicProperties(data)  { 
+        _.each(data, function (x) {
+            var existProperty = _.find(blade.currentEntity.dynamicProperties, function (y) { return y.id == x.id; });
+            if (!angular.isDefined(existProperty)) {
+                blade.currentEntity.dynamicProperties.push(x);
+            }
```

Comment:
Why we can't use here more clear names of variables in functions?
Instead 'x' => might be 'dataItem'
Instead 'y' => can be 'property'
etc